File: src/main/scala/li/cil/oc/integration/thermalexpansion/ModThermalExpansion.scala
package li.cil.oc.integration.thermalexpansion

import li.cil.oc.api.Driver
import li.cil.oc.integration.ModProxy
import li.cil.oc.integration.Mods

object ModThermalExpansion extends ModProxy {
  override def getMod = Mods.ThermalExpansion

  override def initialize() {
    Driver.add(new DriverLamp)
  }
}

File: src/main/scala/li/cil/oc/integration/thermalexpansion/ModThermalExpansion.java
package li.cil.oc.integration.thermalexpansion;

import li.cil.oc.api.Driver;
import li.cil.oc.integration.IMod;
import li.cil.oc.util.mods.Mods;

public final class ModThermalExpansion implements IMod {
    @Override
    public Mods.Mod getMod() {
        return Mods.ThermalExpansion();
    }

    @Override
    public void initialize() {
        Driver.add(new DriverLamp());
    }
}
